.github/workflows/submitProduction.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       - name: Build extension
         uses: actions/setup-node@v1
         with:
-          node-version: "12"
+          node-version: "16.14"
```

Comment:
interestingly - upgrading to the latest stable version of node (18) caused build errors with the old version of Webpack we're using